news/_posts/2020-05-07-productboard-portal-launch.md

```diff
@@ -0,0 +1,100 @@
+subtitle: "Discover PrestaShop's new tool to collect feedback and better prioritize what to build next!"
+date:   2020-05-07 10:00:00
+authors: [ SamuelPires ]
+image:   /assets/images/theme/meta-logo-build.png
+icon: icon-lab
+tags:
+ - contribute
+ - design
+ - feature
+ - news
+ - process
+ - product
+---
+
+As announced [here](https://build.prestashop.com/news/product-features-methodology-at-prestashop), the Core product team recently adopted a new working tool to gather feedback and prioritize features: [Productboard](https://www.productboard.com). No more teasing, it is now time to have a look at how it will look and how we will all be able to handle it!
+
+Productboard makes two things possible for the team:
+
+- On the one hand, having _a dedicated internal interface_ to centralize all feedback, group them into topics, define features, and prioritize them.
+- On the other hand, having _an open public portal_ to explicitly present those features and collect feedback about them. **And that is what this article is about.**
+
+So the team has created this portal called **PrestaShop Feedback** so that anyone can comment and vote for the features on which feedback is needed. Whether you are a merchant, a developer, an agency, a seller on the [Addons Marketplace](http://addons.prestashop.com/), etc. your opinions and suggestions are expected on the [feedback.prestashop.com](https://feedback.prestashop.com) page.
+
+The portal, available in the 'PrestaShop Core' tab, is divided into several sections. The goal of this article is to explain how to use this interface to share your feedback on upcoming features.
+
+
+## How to use PrestaShop Feedback
+
+### See the selected features, express some needs, check their design
+
+Each feature or expression of needs is represented by a card that includes a title and a description, sometimes an image.
+Click on a card to learn more about it.
+
+All cards fall into three sections, each corresponding to a specific step in the product development:
+
+- Under consideration
+- User research
+- Design
+
+![PrestaShop Feedback home](/assets/images/2020/05/productboard-home.png)
+
+#### Under consideration
+Lists the topics and features that have already been identified and structured by the team. At this step, we need feedback to consider as many use cases as possible. It will allow us to draw the most useful feature. Also, rating a card to let the team know how important it is to you will help us prioritize it.
+
+#### User research
+Lists the features the team decided to work on. When we start working on a feature, the first step is always to drive some user research to understand which types of users it may concern, what are the use cases around it, and its purpose. As the goal of user research is to identify the problem or need to solve, the purpose at this point is to get feedback to consider which problems are related to which cases.
```

Comment:
This section lists

Reply:
which types of users may be concerned (moins lourd)

Reply:
```suggestion
This section lists the features the team decided to work on. When we start working on a feature, the first step is always to drive some user research to understand which types of users may be concerned, what are the use cases around it, and its purpose. As the goal of user research is to identify the problem or need to solve, the purpose at this point is to get feedback to consider which problems are related to which cases.
```